studentslist/lib/main.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+final List<int> students = <int>[for(var i=1; i<=100; i+=1) i];
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final title = 'Students List';
+
+    return MaterialApp(
+      title: title,
+      home: Scaffold(
+        appBar: AppBar(title: Text(title)),
+        
+        body: ListView.builder(
+  padding: const EdgeInsets.all(8),
+  itemCount: students.length,
```

Comment:
Please format the file using VSC's formatting tool.

Reply:
done